packages/back-end/src/models/InformationSchemaTablesModel.ts

```diff
@@ -0,0 +1,72 @@
+import omit from "lodash/omit";
+import z from "zod";
+import mongoose from "mongoose";
+import { InformationSchemaTablesInterface } from "../types/Integration";
+import { errorStringFromZodResult } from "../util/validation";
+import { logger } from "../util/logger";
+import { usingFileConfig } from "../init/config";
+
+const informationSchemaTablesSchema = new mongoose.Schema({
+  id: String,
+  organization: {
+    type: String,
+    index: true,
```

Comment:
Instead of indexing on organization, use a compound index on both organization and id. Look at  https://github.com/growthbook/growthbook/blob/main/packages/back-end/src/models/FeatureModel.ts#L81 for an example